packages/truffle/test/scenarios/migrations/fabric-evm.js

```diff
@@ -7,15 +7,15 @@ const Reporter = require("../reporter");
 const sandbox = require("../sandbox");
 const Web3 = require("web3");
 
-describe("migrate with [ @fabric-evm ] interface", function() {
+describe("migrate with [ @fabric-evm ] interface", () => {
```

Comment:
Did you know that Mocha recommends using `function()` instead of `() =>`?

Reply:
🤫